src/python/zquantum/core/wip/estimators/estimation.py

```diff
@@ -0,0 +1,236 @@
+from typing import List, Tuple, Optional
+from openfermion import QubitOperator, IsingOperator
+import pyquil
+import numpy as np
+
+from ...circuit._circuit import Circuit
+from .estimation_interface import (
+    EstimationTask,
+    EstimationTaskTransformer,
+)
+from ...hamiltonian import group_comeasureable_terms_greedy, estimate_nmeas_for_frames
+from ...utils import scale_and_discretize
+from .estimation_interface import EstimationTask
+from ...interfaces.backend import QuantumBackend, QuantumSimulator
+from ...measurement import (
```

Comment:
Did you consider using absolute imports? This way we wouldn't have to change them after we move these files away from the `wip` package.